llama-index-core/llama_index/core/node_parser/relational/markdown_element.py

```diff
@@ -156,7 +158,7 @@ def extract_elements(
                         elements[idx] = Element(
                             id=f"id_{node_id}_{idx}" if node_id else f"id_{idx}",
                             type="table",
-                            element=element,
```

Comment:
I believe this is actually a bug, which I noticed when printing `str(element.element)` the string contained the "type=..., element='...'".